src/components/calculator.js

```diff
@@ -0,0 +1,53 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
[ OPTIONAL ] I recommend avoiding the use of `eslint disable` commands in your projects, as these errors and warnings are typically helpful for identifying areas where you can improve your code. By resolving these issues, you can write more clean and readable code that adheres to industry standards, which can make it easier to maintain your codebase over time. :100:

Reply:
@NahnahAJ Actually package was in the dependencies of my project but still, the linter kept showing errors that the package is not in the dependencies that's why I had to do it
Here is the screenshot of dependencies in `pakage.json`. 
![Capture](https://github.com/batoolfatima2135/math-magicians/assets/61277612/b3511464-1417-4567-ab96-9a8becd544d5)
Let me know if you have a solution to this.